src/planner_test.cc

```diff
@@ -831,6 +831,40 @@ TEST(PlannerTest, DetectOutRangeAcceleration) {
   VerifyCommonExpectations(plantest.segments(), *plantest.GetConfig());
 }
 
+// If we start from zero speed and enqueue a line of segments
+// at constat max feedrate we expect the profile to always be symmetrical.
+TEST(PlannerTest, StraightLine_ProfileSymmetry) {
+  MachineControlConfig *config = new MachineControlConfig();
+  InitTestConfig(config);
+  // The acceleration should stay high enough to avoid making
+  // to avoid numerical issues due to square root calculations
+  // for which the acceleration and deceleration, if they
+  // require too many steps, will end up being very inaccurate.
+  PlannerHarness plantest(0, 0, config);
+  unsigned kNumSteps = 200;
```

Comment:
We're enquening kNumSteps segments, not steps. So wouldn't `kNumSegments` be a slightly better name ?
Also, since it is a constant in our function, should we make it `const` ?

Reply:
(solved): true